_docs/4-kubernetes/2-helloworld.md

```diff
@@ -82,55 +83,54 @@ Please refer to [helloworld](/docs/demo-helloworld.html) for a detailed descript
 1. Lets send all traffic to the v1 version of helloworld:
 
    ```bash
-   $ kubectl create -f examples/k8s-helloworld-default-route-rules.yaml
-   routingrule "set-helloworld-default-v1" created
+   $ a8ctl rule-create -f examples/helloworld-default-route-rules.json
    ```
 
 1. We can confirm the rule is set by running the following command and confirming that `Status.state` is set to `valid`:
 
    ```bash
    $ kubectl get routingrule
-   NAME                        KIND
-   set-helloworld-default-v1   RoutingRule.v1.amalgam8.io
+    NAME                             KIND
```

Comment:
@rshriram @elevran do we just want to get rid of the entire `4-kubernetes` section?  Assuming the docs in `1-demo` cover everything kubernetes related?